src/main/resources/assets/pmmo/lang/fr_fr.json

```diff
@@ -1,393 +1,396 @@
 {
-    "pmmo.afkMultiplierRestored": "En raison de ne plus \u00eatre inactif, votre gain XP est r\u00e9tabli \u00e0 %1$s%%",
```

Comment:
Can we not get the actual characters?  IIRC when we used these escaped codes before it didn't display properly.  We should be able to get the decorated letters with UTF-8.